composer.json

```diff
@@ -8,10 +8,30 @@
             "email": "changeme@mailinator.com"
         }
     ],
-    "autoload" : {
-        "classmap": ["."]
+    "repositories": [
```

Comment:
I had to add this repo because the `pear/Text_Diff` github repo hasn't been published to packagist apparently.